superset-frontend/src/components/DeleteModal/index.tsx

```diff
@@ -52,12 +52,35 @@ export default function DeleteModal({
   title,
 }: DeleteModalProps) {
   const [disableChange, setDisableChange] = useState(true);
+  const [confirmation, setConfirmation] = useState<string>('');
+
+  const hide = () => {
+    setConfirmation('');
+    onHide();
+  };
+
+  const confirm = () => {
+    setConfirmation('');
+    onConfirm();
+  };
+
+  const onChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    const targetValue = event.target.value ?? '';
+    setDisableChange(targetValue.toUpperCase() !== t('DELETE'));
```

Comment:
this is a tricky thing.. so we would need to have a translation for the capitalized text and the lowercase?
I wonder if `t('delete').toUpperCase()` would work.

Reply:
@eschutho this used to be much worse... before it didn't have the `t()` call, so we'd ask people in a different language, _Por favor digite "apagar" para confirmar_, but expected "delete" to unblock the dialog...

Reply:
Ha, yeah I remember when you noticed that. :)